### ISSUE TITLE
Unicode in translations not working

### ISSUE THREAD

**Issue by anjackson** [ukwa/ukwa-pywb#31]
I'm seeing

> Ymweld Ã¢ safle byw a chyfredol
> Edrych os oes copÃ¯au o'r Url hwn gyda archifoedd y we eraill

But the source unicode looks about right: https://github.com/ukwa/ukwa-pywb/blob/b2f8d9e50f189a678cf562c88b468bcbcf2faef9/i18n/translations/cy/LC_MESSAGES/messages.po#L128-L134

So somewhere along the way the locale is breaking things.

**Comment by anjackson** [comment]
Django docs indicate adding `env = LANG=en_US.UTF-8` to `uwsgi.ini` may be required. Trying that now.

**Comment by anjackson** [comment]
No, that didn't help. I don't think nginx is causing the issue, given that server is running in `en_GB.UTF-8`. Not sure how to resolve this.

**Comment by ikreymer** [comment]
A possible fix would be to add <meta charset="utf-8"> tag to the `error.html` and `not_found.html`

```
{% block head %}
<meta charset="utf-8"/>
{% endblock %}
```

The reason is that pywb does not set a charset on the outer frame to allow the browser to run its auto-detection, and these templates are rendered inside the frame. In this way, the browser will detect the charset for pages that don't have it as it would if the pages were not in an iframe.

For whatever reason, the auto-detection here is failing, but luckily we can specify it explicitly!
